backend/clutch-config.yaml

```diff
@@ -34,7 +34,18 @@ modules:
   - name: clutch.module.envoytriage
   - name: clutch.module.k8s
   - name: clutch.module.kinesis
+  - name: clutch.module.topology
```

Comment:
were these changes just for local testing?

Reply:
oops yeah ill remove all my changes here